codalib/util.py

```diff
@@ -250,23 +250,12 @@ def createPREMISEventXML(eventType, agentIdentifier, eventDetail, eventOutcome,
     return eventXML
```

Comment:
It looks like `content` is of type `str`. And I think the code under `if debug`  is not being tested. Maybe add another test similar to `test_raises_exception_without_201_status` with `debug=True`.

Reply:
Are you basing the `content` type based on what it is in a test or from the code itself? The test data types could be affected by mocking. My thought on it being bytes is looking on how `content` is getting it's value in `util.py` and:
```
>>> import urllib.request
>>> response = urllib.request.urlopen('https://library.unt.edu')
>>> type(response.read())
<class 'bytes'>
```
I may be looking at it wrong though. It think it would be good to add a test as you suggest. We have to be careful about not mocking in a way that hides potential problems though.

Reply:
I think you are right. I got it wrong ...`content` is bytes type. I will open the file in bytes mode and write a new test for this piece of code.